src/main/java/com/minecolonies/items/ItemBuildTool.java

```diff
@@ -27,13 +27,12 @@ public ItemBuildTool()
     @Override
     public EnumActionResult onItemUse(ItemStack stack, EntityPlayer playerIn, World worldIn, BlockPos pos, EnumHand hand, EnumFacing facing, float hitX, float hitY, float hitZ)
     {
-        if (!worldIn.isRemote)
+        if (worldIn.isRemote)
         {
-            return EnumActionResult.FAIL;
+            playerIn.addStat(ModAchievements.achievementWandOfbuilding);
```

Comment:
It did not, after looking at the code I'm not sure it ever did as the server would have exited the method before it was called.  I moved it to the top of the method to make sure it is called on use.